### ISSUE TITLE
Background class is predicted as one of foreground classes

### ISSUE THREAD

**Issue by DISAPPEARED13** [VainF/DeepLabV3Plus-Pytorch#102]
Hi, there, I use Deeplabv3+ as a multiple class semantic segmentation, and background class is regarded as label 0, while the other is from 1 to 13.
But in validation, the background class is predicted as class 4. It's really confusing.

some training details:
SGD optimizer
learning rate is 0.001
data preprocess issue is refered to nnunet.
loss: only Dice loss can work, while others were collapsed.

Thanks! Really thanks a lot!

**Comment by DISAPPEARED13** [comment]
I fixed this problem. sorry to bother. Thanks a lot!
